src/components/Insights/HeatMap.js

```diff
@@ -18,337 +18,356 @@ import 'leaflet.markercluster/dist/MarkerCluster.Default.css';
+
 export const HeatMap = React.createClass({
   mixins: [FluxMixin, StoreWatchMixin],
-  
-  getInitialState(){
-      return{
-          mapProgressPercent: -1,
-          intervalTime: 200,
-          selectedTileId: false,
-          modalTitle: ''
-      };
+
+  getInitialState() {
+    return {
+      mapProgressPercent: -1,
+      intervalTime: 200,
+      selectedTileId: false,
+      modalTitle: ''
+    };
   },
-  
+
   getStateFromFlux() {
     return this.getFlux().store("DataStore").getState();
   },
-  
-  addInfoBoxControl(){
-      let info = L.control();
-      
-      if(this.map){
-          info.onAdd = map => {
-			this._sentimentDiv = L.DomUtil.create('div', 'info');
-			info.update();
-			return this._sentimentDiv;
-		  };
-          
-          info.options = {
-            position: 'topleft'  
-          };
-
-		  info.update = props => {
-            let sentimentLevel = props && props.weightedSentiment ? props.weightedSentiment : 0;
-            let sentimentClass = "", sentimentIcon = "";
-
-            if(sentimentLevel < 30){
-                sentimentClass = "positiveSenitment";
-                sentimentIcon = "sentiment_very_satisfied";
-            }else if(sentimentLevel < 55){
-                sentimentClass = "neutralSentiment";
-                sentimentIcon = "sentiment_neutral";
-            }else if(sentimentLevel < 80){
-                sentimentClass = "neutralNegativeSentiment";
-                sentimentIcon = "sentiment_dissatisfied";
-            }else{
-                sentimentClass = "negativeSenitment";
-                sentimentIcon = "sentiment_very_dissatisfied";
-            }
 
-            let sentimentIconSpan = `<span class="material-icons sentimentIcon ${sentimentClass}Icon" color="#ffffff">${sentimentIcon}</span>`;
-            let infoHeaderText = "<h5>Sentiment Summary</h5>";
-            let infoBoxInnerHtml = 
-            `<div id="sentimentGraph">
-                <div class="sentimentGraphBar ${sentimentClass}">
-                    ${parseFloat((sentimentLevel / 100) * 10).toFixed(2)} / 10
-                </div>
-                ${sentimentIconSpan}
-            </div>`;
-            
-			this._sentimentDiv.innerHTML = infoHeaderText + infoBoxInnerHtml;
-		  };
-
-		  info.addTo(this.map);
-          this.sentimentGraph = info;
-      }
+  addInfoBoxControl() {
+    let info = L.control();
+
+    if (this.map) {
+      info.onAdd = map => {
+        this._sentimentDiv = L.DomUtil.create('div', 'info');
+        info.update();
+        return this._sentimentDiv;
+      };
+
+      info.options = {
+        position: 'topleft'
+      };
+
+      info.update = props => {
+        let sentimentLevel = props && props.weightedSentiment ? props.weightedSentiment : 0;
+        let sentimentClass = "";
+        let sentimentIcon = "";
+
+        if (sentimentLevel < 30) {
```

Comment:
Yeah, I'll make a second pass which will focus on extracting constants/methods. Wanted to keep the first round minimal to make reviewing easier (whitespace changes mess up diffs!).